week-4/InClass/C-more-complex-objects/exercise-3.js

```diff
@@ -33,10 +33,10 @@ let parkAvenueHouse = {
 
 // returns the full name (first name + last name) of the owner of the house
 function getOwnerFullName(house) {
-  //let fullName = house.currentOwner.firstName + ' ' + house.currentOwner.lastName;
-  //fullName = house.currentOwner.firstName + ' ' + house.currentOwner.lastName;
-  //return house.currentOwner.firstName + ' ' + house.currentOwner.lastName;  // concatenation
-  return `${house.currentOwner.firstName} ${house.currentOwner.lastName}` // string interpolation
+  //let fullName = house.currentOwner.firstName + ' ' + house.currentOwner.lastName; // dont need to use let 
+  //fullName = house.currentOwner.firstName + ' ' + house.currentOwner.lastName; //  or new variable name
+  //return house.currentOwner.firstName + ' ' + house.currentOwner.lastName;  // concatenation example returns dotnotation objectName and the key:value properties eg.nestedArray.firstName and lastName
+  return `${house.currentOwner.firstName} ${house.currentOwner.lastName}` // string interpolation, the space between the stringsinterpolation seperates the dotnation properties

```

Comment:
nice, neat code and the comments are also helpful for you to look back on, and others who may come across your code too. Great!